connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -409,8 +457,34 @@ public static IdToken getIdToken() {
      */
     public static void getUserInfo(Callback<UserInfo> userInfoCallback) {
         Validator.sdkInitialized();
-        sConnectIdService.getUserInfo(userInfoCallback);
+        sdkProfile.getConnectIdService().getUserInfo(userInfoCallback);
     }
 
+    /**
+     * MobileConnect stuff
+     */
+
+    public static synchronized void sdkInitializeMobileConnect(
+            Context context,
+            OperatorDiscoveryConfig operatorDiscoveryConfig) {
+        if (isInitialized()) {
+            return;
+        }
+        sdkProfile = new MobileConnectSdkProfile(
+                context,
+                operatorDiscoveryConfig,
+                fetchBooleanProperty(getApplicationInfo(context), USE_STAGING_PROPERTY),
+                fetchBooleanProperty(getApplicationInfo(context), CONFIDENTIAL_CLIENT_PROPERTY));
+        context.registerReceiver(
```

Comment:
Ah, sorry, might have misunderstood that one a bit.
The dev can currently choose which context to use, so it might be an application context or it might be an activity context. In the latter case I would expect it to die if the activity dies.